app/controllers/prospects_controller.rb

```diff
@@ -0,0 +1,65 @@
+class ProspectsController < ApplicationController
+  skip_before_action :authenticate_user!
```

Comment:
This feels like a hack, I added these lines so I could progress and make prospects page. I need to know more about policies.